cuda_pathfinder/cuda/pathfinder/_dynamic_libs/supported_nvidia_libs.py

```diff
@@ -384,12 +395,91 @@
         "nvvm70.dll",
     ),
 }
+SUPPORTED_WINDOWS_DLLS_OTHER = {
+    "mathdx": ("mathdx64_0.dll",),
+}
+SUPPORTED_WINDOWS_DLLS = SUPPORTED_WINDOWS_DLLS_CTK | SUPPORTED_WINDOWS_DLLS_OTHER
 
 LIBNAMES_REQUIRING_OS_ADD_DLL_DIRECTORY = (
     "cufft",
     "nvrtc",
 )
 
+LIBNAMES_REQUIRING_RTLD_DEEPBIND = ("cufftMp",)
+
+# Based on output of toolshed/make_site_packages_libdirs_linux.py
+SITE_PACKAGES_LIBDIRS_LINUX_CTK = {
+    "cublas": ("nvidia/cu13/lib", "nvidia/cublas/lib"),
+    "cublasLt": ("nvidia/cu13/lib", "nvidia/cublas/lib"),
+    "cudart": ("nvidia/cu13/lib", "nvidia/cuda_runtime/lib"),
+    "cufft": ("nvidia/cu13/lib", "nvidia/cufft/lib"),
+    "cufftw": ("nvidia/cu13/lib", "nvidia/cufft/lib"),
+    "cufile": ("nvidia/cu13/lib", "nvidia/cufile/lib"),
+    # "cufile_rdma": ("nvidia/cu13/lib", "nvidia/cufile/lib"),
+    "curand": ("nvidia/cu13/lib", "nvidia/curand/lib"),
+    "cusolver": ("nvidia/cu13/lib", "nvidia/cusolver/lib"),
+    "cusolverMg": ("nvidia/cu13/lib", "nvidia/cusolver/lib"),
+    "cusparse": ("nvidia/cu13/lib", "nvidia/cusparse/lib"),
+    "nppc": ("nvidia/cu13/lib", "nvidia/npp/lib"),
+    "nppial": ("nvidia/cu13/lib", "nvidia/npp/lib"),
+    "nppicc": ("nvidia/cu13/lib", "nvidia/npp/lib"),
+    "nppidei": ("nvidia/cu13/lib", "nvidia/npp/lib"),
+    "nppif": ("nvidia/cu13/lib", "nvidia/npp/lib"),
+    "nppig": ("nvidia/cu13/lib", "nvidia/npp/lib"),
+    "nppim": ("nvidia/cu13/lib", "nvidia/npp/lib"),
+    "nppist": ("nvidia/cu13/lib", "nvidia/npp/lib"),
+    "nppisu": ("nvidia/cu13/lib", "nvidia/npp/lib"),
+    "nppitc": ("nvidia/cu13/lib", "nvidia/npp/lib"),
+    "npps": ("nvidia/cu13/lib", "nvidia/npp/lib"),
+    "nvJitLink": ("nvidia/cu13/lib", "nvidia/nvjitlink/lib"),
+    "nvblas": ("nvidia/cu13/lib", "nvidia/cublas/lib"),
+    "nvfatbin": ("nvidia/cu13/lib", "nvidia/nvfatbin/lib"),
+    "nvjpeg": ("nvidia/cu13/lib", "nvidia/nvjpeg/lib"),
+    "nvrtc": ("nvidia/cu13/lib", "nvidia/cuda_nvrtc/lib"),
+    "nvvm": ("nvidia/cu13/lib", "nvidia/cuda_nvcc/nvvm/lib64"),
```

Comment:
oh man, so in CUDA 13 the `nvvm` subdir still exists, but only libdevice is still kept there while everything else is moved to bin/include/lib... 😞

Reply:
This is annoying... so the wheel layout for NVVM is changed, but not the system CTK which still has `$CUDA_PATH/nvvm`...
```shell
$ tree /usr/local/cuda-13.0/nvvm/
/usr/local/cuda-13.0/nvvm/
├── bin
│   └── cicc
├── include
│   └── nvvm.h
├── lib64
│   ├── libnvvm.so -> libnvvm.so.4
│   ├── libnvvm.so.4 -> libnvvm.so.4.0.0
│   └── libnvvm.so.4.0.0
└── libdevice
    └── libdevice.10.bc

4 directories, 6 files
```